applications/dashboard/models/class.tagmodel.php

```diff
@@ -530,18 +532,22 @@ public function getDiscussions($tag, $limit, $offset) {
             ->join('Tag t', 't.TagID = td.TagID')
             ->whereIn('t.Name', $tags)
             ->limit($limit, $offset)
+            ->orderBy($sortField, $sortDirection)
             ->get()->resultArray();
 
         $taggedDiscussionIDs = array_column($taggedDiscussionIDs, 'DiscussionID');
 
         $discussionModel = new DiscussionModel();
-        $discussions = $discussionModel->get(
-            0,
-            '',
+        $discussions = $discussionModel->getWhere(
             [
                 'Announce' => 'all',
                 'd.DiscussionID' => $taggedDiscussionIDs,
-            ]
+            ],
+            $sortField,
+            $sortDirection,
+            '',
+            0
+
```

Comment:
Do we need these last two parameters? You have an empty string for `$limit` and a zero for `$offset. If you omit both of those values, does it still function the same way?

Reply:
An empty line has snuck into your parameters!

Reply:
Those can be omitted.